nss-day-cohort-43-csharp/MVC-Silver-Snakes
# Se view comments

# Description
Posts now have a view all Comments section. Users can add comments to a post.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
- Go to a post's comment view
- Add more than one comment
- See if they show in list
# Checklist:
